docs/BREAKING_CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Breaking Changes
 
+## 0.47.0
+
+The function `SchemaUtils.checkExcessiveIndices` used to check both excessive indices and excessive foreign key constraints. It now has a different behaviour and
+deals with excessive indices only. A new function, `SchemaUtils.checkExcessiveForeignKeyConstraints`, deals with excessive foreign key constraints.
+
```

Comment:
It's not as important since we're going from Unit to List (and not vice versa), but I'd still mention that the functions also now return something in addition to logging.